.github/workflows/CI.yml

```diff
@@ -573,6 +574,44 @@ jobs:
           #    docker tag $IMAGE_NAME $REPO_NAME:$TAG
           #    docker push $REPO_NAME:$TAG
 
+  e2etest-public-ecr-release:
```

Comment:
Why can't this just be done in the `upload to ECR` step above?

Reply:
Testing multiple docker logins within the same step may be a problem and I'm not sure how to tests this without affecting the current repo for prod. With them not in the same job now they can also run in parallel which will reduce ci time.